Homework Lesson 16/Task 3.py

```diff
@@ -0,0 +1,55 @@
+class Product:
+    def __init__(self, type, name, price):
+        self.type = type
+        self.name = name
+        self.price = price
+
+class ProductStore:
+    def __init__(self):
+        self.products = {}
+        self.income = 0
+
+    def add(self, product, amount):
+        if product.name not in self.products:
+            self.products[product.name] = {
+                'product': product,
+                'amount': amount,
+                'price': product.price * 1.3
```

Comment:
why 1.3?)

Reply:
The task says "add(product, amount) - adds a specified quantity of a single product with a predefined price premium for your store(30 percent)"